api/src/Entity/Camp.php

```diff
@@ -113,7 +113,11 @@ class Camp extends BaseEntity implements BelongsToCampInterface, CopyFromPrototy
     /**
      * All the progress labels within this camp.
      */
-    #[ApiProperty(writable: false, example: '["/progress_labels/1a2b3c4d"]')]
+    #[ApiProperty(
```

Comment:
I'm a bit surprised this didn't break `ReadCampTest`. Shouldn't this change the response of the Camp entity?

Reply:
This is not verified.
No test, no merci.
But it changes the response, see here: https://github.com/ecamp/ecamp3/pull/6834/files#r1970498920